transactions/randomBeaconHistory/record_source_of_randomness.cdc

```diff
@@ -0,0 +1,22 @@
+import "RandomBeaconHistory"
+
+/// Commits the source of randomness for the requested block height from the RandomBeaconHistory Heartbeat
+///
+/// Note: commits to RandomBeaconHistory.Heartbeat will be completed in the system chunk transaction by the service
+/// account. This transaction is included for initial testing purposes until randomSourceHistory() is available in
+/// an emulator build
+///
+transaction(sor: [UInt8]) {
```

Comment:
I think we can remove this transaction. Looks like it's included in [system chunk transaction](https://github.com/onflow/flow-go/pull/4582/files#diff-73e7c4fe793765e3190e62aecdfd6e14b10d5226f61d26a56551192e4df7d301R16)